dbt/task/compile.py

```diff
@@ -6,16 +6,44 @@
 import dbt.project
 from collections import defaultdict
 
+from functools import partial
+import networkx as nx
+
 CREATE_STATEMENT_TEMPLATE = """
 create {table_or_view} {schema}.{identifier} {dist_qualifier} {sort_qualifier} as (
     {query}
 );"""
 
+class Linker(object):
```

Comment:
this object is just the dependency graph now, I think it would be better to call it something more direct... like... `DepGraph` or similar.

FWIW, I also think it would be totally fine to not create our own object for this, and instead pass around an `nx.DiGraph` object directly, using some helper functions in a dedicated namespace... but I am baking a lot of my anti-OOP biases into that recommendation :)